src/paths.md

```diff
@@ -62,7 +62,10 @@ mod m {
 > &nbsp;&nbsp; | [_SimplePathSegment_]
 >
 > _GenericArgsBinding_ :\
-> &nbsp;&nbsp; [IDENTIFIER] `=` [_Type_]
+> &nbsp;&nbsp; [IDENTIFIER] _GenericArgs_<sup>?</sup> `=` [_Type_]
```

Comment:
For the record, only thing this adds is GenericArgs<sup>?</sup>, so you can write `T: Trait<Assoc<i32> = Ty>`